pkgs/development/python-modules/pwntools/default.nix

```diff
@@ -2,9 +2,7 @@
 , buildPythonPackage
 , debugger
 , fetchPypi
-, isPy3k
 , Mako
-, makeWrapper
```

Comment:
Mmmh, I tried and it works, but I do not understand why...
I thought you needed to have `makeWrapper` passed to the `nativeBuildInputs` to be able to use it in `postFixup`...

Reply:
According to https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-functions it seems to be included in the standard build environment, but I've also seen a lot of places import it manually so I don't if that was always the case. In this case it wasn't even given to `nativeBuildInputs`, which is why the PR review bot detected it as dead code.

Reply:
I don't think it is. But it is in Python's.